examples/preprocessing/eog_regression.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+"""
+=======================================
+Reduce EOG artifacts through regression
+=======================================
+
+Reduce artifacts by regressing the EOG channels onto the rest of the channels
+and then subtracting the EOG signal.
+
+This is a quick example to show the most basic application of the technique.
+See the :ref:`tutorial <tut-artifact-regression>` for a more thorough
+explanation that demonstrates more advanced approaches.
+
+Authors: Marijn van Vliet <w.m.vanvliet@gmail.com>
+
+License: BSD (3-clause)
+"""
```

Comment:
I think a blank line is missing, because the following lines are not rendered as code.